src/wre/game.ts

```diff
@@ -1,5 +1,7 @@
 import { Execution, Natives, RuntimeObject, RuntimeValue } from '../interpreter/runtimeModel'
 
+const runtimePosition = '<position>'
```

Comment:
"Runtime" no me convence del todo porque todas son "calculadas en runtime". Esto sería una posición que s emete desde afuera del objeto (a diferencia de las _comunes_ que se le pregunta al objeto).

```suggestion
const externalPosition = '<position>'
```

Reply:
De acuerdo. Ya lo cambiamos.

Reply:
NIT: Si no les jode estaría bueno mantener la convención de que las constantes se escriban en mayusculas. Por otro lado a la gente que viene de afuera ni runtime ni external nos dice mucho de qué es eso... Yo sería más verboso e iría por `VISUAL_POSITION_FIELD_NAME`.

Reply:
Perdón, no habíamos prestado atención a la convención. Lo tenemos en cuenta para el futuro. Igual estuvimos hablando con @PalumboN y al final vamos a ir por el camino de deprecar los métodos `addVisualIn` y `addVisualCharacterIn` como se había acordado en este [issue](https://github.com/uqbar-project/wollok-language/issues/57). La idea sería cerrar los PRs relacionados ([este](https://github.com/uqbar-project/wollok-run-client/pull/64) y [este otro](https://github.com/uqbar-project/wollok-language/pull/110)) para después hacer otros nuevos quitando dichos métodos y cambiando la estrategia para conocer la posición de los visuales. Ahora sólo enviaríamos el mensaje `position()`.

Reply:
Nah, no pasa nada igual es una boludez.
+1 a volar los métodos horribles esos.